### PR TITLE
OnDie documentation

### PR DESCRIPTION
**Pull request by johneasterday**

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -83,6 +83,28 @@ http_proxy_port: 900
...
+ondie_autoupdate property is set to true.
+
+`ondie_cache`: specifies the path to the directory containing the OnDie certs and CRLs
+               (required if supporting OnDie, optional otherwise)
+`ondie_autoupdate`: if "true" then the OnDie certs and CRLs are downloaded from the cloud at start up into the directory specified by ondie_cache.
+                    Note that this requires internet access by the component. Should a component be run in on-prem mode then this setting should be set
+                    to "false". In such cases, the artifacts can be preloaded by running the script in component-sample/scripts/onDieScript.py when
+                    access is available or from another machine with access and then copied into the ondie_cache directory.
+                    (optional, default = false)
+`ondie_zip_artifact`: specifies the URL containing of the zip file that contains the OnDie certs and CRLs
```
[the remark this answers is not in the document]
Comment (johneasterday) [reply demo/README.md]: fixed formatting

File: demo/README.md [hunk 2]
```diff
@@ -83,6 +83,28 @@ http_proxy_port: 900
...
+                     (optional, default = https://tsci.intel.com/content/csme.zip)
+`ondie_check_revocations`: if "true" then revocations are checked by the component, no revocation checking is done if "false".
+                           (optional, default = true for Manufacturer and Owner, false for RV)
+
```
[the remark this answers is not in the document]
Comment (johneasterday) [reply demo/README.md:100]: added notes to individual component README files referencing this main README for OnDie.

File: demo/manufacturer/README.md [hunk 3]
```diff
@@ -74,6 +74,10 @@ Some required runtime arguments
 
   Default value: MfgApiPass123
 
+## Support for OnDie devices
+
+Refer to [Demo README](demo/README.md) for steps to configure manufacturer to support OnDie devices.
```
Comment (Darshini-Parikh93) [inline demo/manufacturer/README.md:79]: The link should be (../README.md) in place of (demo/README.md). The current links for all 3 of manufacturer, owner and rv is resulting in 404 Not found error.

Comment (johneasterday) [reply demo/manufacturer/README.md]: oops, fixed the links

**Review by Darshini-Parikh93 — APPROVED** [review]

**Review by trbehera — APPROVED** [review]

**Pull request merged**